db/dcrpg/queries.go

```diff
@@ -610,8 +610,10 @@ func InsertTickets(db *sql.DB, dbTxns []*dbtypes.Tx, txDbIDs []uint64, checked,
 			if len(tx.Vouts[0].ScriptPubKeyData.Addresses) > 0 {
 				stakesubmissionAddress = tx.Vouts[0].ScriptPubKeyData.Addresses[0]
 			}
-			scriptSubClass, _ := txscript.GetStakeOutSubclass(tx.Vouts[0].ScriptPubKey, false) // only looking for multisig, so treasury types are irrelevant
-			isMultisig = scriptSubClass == txscript.MultiSigTy                                 // err return nonstandard, so false
+			isMultisig = stdscript.IsStakeSubmissionScriptHashScript(tx.Vouts[0].Version, tx.Vouts[0].ScriptPubKey)
+			// This was historically broken, always setting isMultisig false! A
```

Comment:
While it's not an entirely unreasonable guess given actual usage patterns, it's perhaps worth point out that this is still not really accurate, since the only you could know is to retroactively set it when the ticket is redeemed to first determine that this output is a p2sh as you're doing here and then also look to see if the signature script redeeming the output is `stdscript.IsMultiSigSigScriptV0`.

Reply:
Indeed, it was ill-conceived with the assumption that the scripthash script was itself multi-sig.  Considering the historical 1-of-2 stake pool tickets, that made sense, but it's generally a bad inference.  The bool and the flag in the database would have been better as isP2SH or something like that.  But as I say in the comment, it has never been flagging anything right, and clearly it's not used, so I'm inclined to drop this column of the tickets table.